src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageDetailControlModel.cs

```diff
@@ -339,6 +339,14 @@ public bool IsProjectPackageReference
             }
         }
 
+        public bool IsFloatingVersionSupported
```

Comment:
Looking at the commit list, GitHub seems to be a bit confused and seems to have included some dev branch commits. Therefore, this change isn't part of the static graph restore changes, right?

I'm not sure what changes are part of the PR and what aren't 😞

Reply:
Sorry I had tried rebasing on different commits to see if any recent changes in our repo had caused the compiler error we're seeing.  I've rebased again on top of `dev` so the commit list is correct again.